examples/complete/main.tf

```diff
@@ -61,28 +59,32 @@ module "cos_bucket1" {
   source                             = "../../"
   resource_group_id                  = module.resource_group.resource_group_id
   region                             = var.region
+  cross_region_location              = null
   cos_instance_name                  = "${var.prefix}-cos"
   cos_tags                           = var.resource_tags
   bucket_name                        = "${var.prefix}-bucket-1"
   existing_key_protect_instance_guid = module.key_protect_all_inclusive.key_protect_guid
   key_protect_key_crn                = module.key_protect_all_inclusive.keys["${local.key_ring_name}.${local.key_name}"].crn
   sysdig_crn                         = module.observability_instances.sysdig_crn
-  activity_tracker_crn               = module.observability_instances.activity_tracker_crn
+  activity_tracker_crn               = var.activity_tracker_crn
```

Comment:
you will need a local variable now with ternary syntax to either use `var.activity_tracker_crn` value if passed in, or otherwise use the CRN of the AT instance that will be created by the example